.github/workflows/test-e2e.yml

```
@@ -98,13 +98,27 @@ jobs:
             make ui-test-e2e-local
           fi
+        # Deliberately tolerated here so the artifact upload below still runs on
+        # a failing suite; the verdict is enforced by the gate at the end of the
+        # job, which is the step that decides the job's conclusion.
         continue-on-error: true
-    
+
       - name: Upload Test Artifacts
         uses: ./.github/actions/upload-test-artifacts
         with:
           include-pr-metadata: ${{ github.event_name == 'pull_request' }}
```

actionlint (1.7.12)

[error] input "include-pr-metadata" is not defined in action "Upload Test Artifacts" defined at "./.github/actions/upload-test-artifacts". available inputs are 

(action)